lint

```diff
@@ -145,6 +145,18 @@ lint_sh() {
     return $lint_result
 }
 
+lint_tf() {
+    filename="$1"
+    local lint_result=0
+
+    if [ -n "$(diff <(hclfmt "${filename}") "${filename}")" ]; then
```

Comment:
this is a minor nit, and is again copied from how the other functions do it, but I'd rather use the exit status of `diff` than `[ -n $(diff) ]` to determine if two files differ.